resources/check.rb

```diff
@@ -3,7 +3,7 @@
 attribute :type, :kind_of => String, :equal_to => %w[status metric]
 attribute :command, :kind_of => String, :required => true
 attribute :timeout, :kind_of => Integer
-attribute :subscribers, :kind_of => Array
+attribute :subscribers, :kind_of => Array, :required => true
```

Comment:
Making this attribute required conflicts with the validation logic you've added below. I don't think it makes sense to add this.